The 'update playwright snapshots' action fails

See for instance [this run](https://github.com/mwouts/jupytext/actions/runs/12338326838/job/34433142072):
```bash
Run hub api repos/mwouts/jupytext/issues/comments/2543833016/reactions --raw-field 'content=+1'
/home/runner/work/_temp/70dbc7db-7468-45a7-934e-225fd75776f9.sh: line 1: hub: command not found
Error: Process completed with exit code 127.
```

--- a/.github/workflows/update-playwright-snapshots.yml
+++ b/.github/workflows/update-playwright-snapshots.yml
@@ -16,7 +16,7 @@ jobs:
     steps:
       - name: React to the triggering comment
         run: |
-          hub api repos/${{ github.repository }}/issues/comments/${{ github.event.comment.id }}/reactions --raw-field 'content=+1'
+          gh api repos/${{ github.repository }}/issues/comments/${{ github.event.comment.id }}/reactions --raw-field 'content=+1'
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
@@ -25,9 +25,7 @@ jobs:
 
       - name: Checkout the branch from the PR that triggered the job
         run: |
-          # PR branch remote must be checked out using https URL
-          git config --global hub.protocol https
-          hub pr checkout ${{ github.event.issue.number }}
+          gh pr checkout ${{ github.event.issue.number }}
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 

